linux/utils/install_uninstall_package.yml

```diff
@@ -111,11 +111,30 @@
           when: guest_os_ansible_pkg_mgr | lower == "yum"
 
         - name: "{{ package_manage_op }} packages on {{ guest_os_ansible_distribution }} {{ guest_os_ansible_distribution_ver }}"
-          ansible.builtin.dnf:
-            name: "{{ package_manage_list }}"
-            allow_downgrade: true
-            state: "{{ package_state }}"
-          delegate_to: "{{ vm_guest_ip }}"
+          block:
+            - name: "Get DNF version on {{ guest_os_ansible_distribution }} {{ guest_os_ansible_distribution_ver }}"
+              ansible.builtin.command: "dnf --version | head -n 1 | cut -d '.' -f 1"
```

Comment:
use `ansible.builtin.shell` instead of `ansible.builtin.command` when command has pipe